### PR TITLE
Release/0.1.0

### PR DESCRIPTION
**Pull request by SencyCaspit**
Demo App is ready

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,130 @@
-# smkit-ui-flutter-demo
+# SMKitUI SDK - Demo App
 
-Hold tight! This demo will soon be ready.
+### [Usage](#usage)
+* [Initialization](#initialization)
+* [Android](#Android)
 
-Can't wait? [contact us](mailto:support@sency.ai) and let us know.
+### [API](#usage)
```
Comment (ofergoldstein) [inline README.md]: @SencyCaspit both links are to the same location #usage 
that's probably a mistake

File: README.md [hunk 2]
```diff
@@ -1,7 +1,130 @@
...
+## API
+### Start Assessment
+Start the Assessment screen. In the parameters method `startAssessment()` You can add `SencyHandlerStatus` to track the success of the method. 
+You need to process the result that the methods return in the callback.
+
+Please make sure you call `startAssessment()` only after configuring the plugin with your "AUTH_KEY".
+```dart
+  // Under Button Widget of sort
+  onPressed: () {
+      _smkitUiFlutterPlugin.startAssessment();
```
Comment (ofergoldstein) [inline README.md]: @SencyCaspit I'm missing the part where we handle result data (error / success)

File: README.md [hunk 3]
```diff
@@ -1,7 +1,130 @@
...
+      getInFrame: null, // =>  getInFrame: string | null (url for a sound)
+      bodycalFinished: null, // =>  bodycalFinished: string | null (url for a sound)
+      workoutClosure: null, // =>  workoutClosure: string | null (url for a sound)
+    );
+}
+
+// Under Button widget of sort 
+onPressed: () {
+    _smkitUiFlutterPlugin.startCustomWorkout(
+        workout: getDemoWorkout(),
```
Comment (ofergoldstein) [inline README.md]: @SencyCaspit same here

File: android/app/build/generated/source/buildConfig/debug/com/example/smkit_ui_flutter_plugin_example/BuildConfig.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
+/**
+ * Automatically generated file. DO NOT MODIFY
+ */
+package com.example.smkit_ui_flutter_plugin_example;
+
+public final class BuildConfig {
+  public static final boolean DEBUG = Boolean.parseBoolean("true");
+  public static final String APPLICATION_ID = "com.example.smkit_ui_flutter_plugin_example";
```
Comment (ofergoldstein) [inline android/app/build/generated/source/buildConfig/debug/com/example/smkit_ui_flutter_plugin_example/BuildConfig.java]: @SencyCaspit maybe its better to use "com.sency.smkit_ui_flutter_plugin_example"

File: android/build/kotlin/kaptGenerateStubsDebugKotlin/cacheable/last-build.bin
Comment (ofergoldstein) [inline android/build/kotlin/kaptGenerateStubsDebugKotlin/cacheable/last-build.bin:1]: @SencyCaspit do we actually need git to track all these binary files??

Comment (SencyCaspit) [reply android/build/kotlin/kaptGenerateStubsDebugKotlin/cacheable/last-build.bin:1]: Fixed.

File: README.md (hunk 1 above)
Comment (SencyCaspit) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (SencyCaspit) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (SencyCaspit) [reply README.md]: Fixed.

**Review by ofergoldstein — APPROVED** [review]

**Pull request merged**